lib/flutter_label_printer_method_channel.dart

```diff
@@ -163,4 +163,117 @@ class MethodChannelFlutterLabelPrinter extends FlutterLabelPrinterPlatform {
 
     return result ?? false;
   }
+
+  @override
+  Future<bool> setPageWidthHMA300L(int width) async {
+    final result = await methodChannel.invokeMethod<bool>(
+      'hk.gogovan.label_printer.setPageWidthHMA300L',
+      <String, dynamic>{
+        'width': width,
+      },
+    );
+
+    return result ?? false;
+  }
+
+  @override
+  Future<bool> setAlignHMA300L(int align) async {
+    final result = await methodChannel.invokeMethod<bool>(
+      'hk.gogovan.label_printer.setAlignHMA300L',
```

Comment:
better extract all function names as constants @luiges90

Reply:
These codes are shared between Dart layer and Native layer. Constants cannot be shared across this gap.